framework/src/main/java/org/tron/core/net/TronNetService.java

```diff
@@ -107,6 +111,7 @@ protected void onMessage(PeerConnection peer, TronMessage msg) {
           throw new P2pException(TypeEnum.NO_SUCH_MESSAGE, msg.getType().toString());
       }
     } catch (Exception e) {
+      monitorMetric.getCounter(MonitorMetric.NET_ERROR_PROTO_COUNT).inc();
```

Comment:
This will collect all message exceptions, like TCP, UDP.   However,  some http request would give wrong format of contract or transactionID and so on, this will not be covered  by this piece of code.